### PR TITLE
APP-5305: New response body API

### PR DESCRIPTION
**Pull request by AramSemerjyan**

### CODE REVIEW

File: Sources/PayseraMokejimaiSDK/Entities/Tax/PSTaxInformation.swift [hunk 1]
```diff
@@ -1,17 +1,13 @@
...
     
     public init() {}
     
     required public init?(map: Map) {}
     
     public func mapping(map: Map) {
-        message         <- map["message"]
-        messageCode     <- map["message_code"]
-        tins            <- map["tins"]
+        messages <- map["messages"]
```
Comment (MohsenKhosravinia) [inline Sources/PayseraMokejimaiSDK/Entities/Tax/PSTaxInformation.swift:11]: align it please

Comment (AramSemerjyan) [reply Sources/PayseraMokejimaiSDK/Entities/Tax/PSTaxInformation.swift:11]: align to what? there is no other property here except messages

Comment (MohsenKhosravinia) [reply Sources/PayseraMokejimaiSDK/Entities/Tax/PSTaxInformation.swift:11]: the `<- map["messages]` with upper ones.

Comment (MohsenKhosravinia) [reply Sources/PayseraMokejimaiSDK/Entities/Tax/PSTaxInformation.swift:11]: oh sorry. my bad

**Pull request merged**